src/main/java/seedu/address/model/person/ApplicationStatus.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model.person;
+
+public class ApplicationStatus {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Application Status should be Pending, Accepted or Rejected";
+    public static final String PENDING_STATUS = "Pending";
+    public static final String ACCPETED_STATUS = "Accepted";
+    public static final String REJECTED_STATUS = "Rejected";
+
+    private String statusType;
+
+    /**
+     * Constructor for Application Status.
+     *
+     * @param statusType the string value of the status.
+     */
+    public ApplicationStatus(String statusType) {
+        String input = statusType.toLowerCase();
+        if (input.equals(PENDING_STATUS.toLowerCase())) {
+            this.statusType = PENDING_STATUS;
+        } else if (input.equals(ACCPETED_STATUS.toLowerCase())) {
+            this.statusType = ACCPETED_STATUS;
+        } else if (input.equals(REJECTED_STATUS.toLowerCase())) {
+            this.statusType = REJECTED_STATUS;
```

Comment:
Perhaps could have made use of `isValidStatus(statusType)` instead?

Reply:
I think for this particular class might not be able to make use for the `isValidStatus(statusType)` method because this requires the assigning of the variable in the class! The other method would probably be more useful for the test cases.